src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -41,6 +41,7 @@ class FunSpec private constructor(builder: Builder) {
   val parameters = builder.parameters.toImmutableList()
   val exceptions = builder.exceptions.toImmutableList()
   val code = builder.code.build()
+  val isSingleExpression = builder.isSingleExpression
```

Comment:
I don't think "single" adds any value here. Kotlin calls these "expression bodies" so I think either `isExpression` or `isExpressionBody` is sufficient. I would also think we rename `code` to `body` here...